src/components/apps/PublishAppDialog.js

```diff
@@ -0,0 +1,409 @@
+                            onClose={handleClose}
+                            baseId={parentId}
+                            title={t("publicSubmissionTitle", {
+                                appName: app.name,
+                            })}
+                            actions={
+                                <>
+                                    <Button
+                                        id={build(parentId, ids.CANCEL_BTN)}
+                                        onClick={handleClose}
+                                    >
+                                        {i18nCommon("cancel")}
+                                    </Button>
+                                    <Button
+                                        id={build(parentId, ids.SUBMIT_BTN)}
+                                        type="submit"
+                                        color="primary"
+                                        onClick={handleSubmit}
+                                    >
+                                        {i18nCommon("submit")}
+                                    </Button>
+                                </>
+                            }
+                        >
+                            {requestStatus === constants.LOADING && (
+                                <CircularProgress
+                                    size={30}
+                                    thickness={5}
+                                    style={{
+                                        position: "absolute",
+                                        top: "50%",
+                                        left: "50%",
+                                    }}
+                                />
+                            )}
+                            {error && (
+                                <ErrorTypographyWithDialog
+                                    errorMessage={t("publicationRequestFailed")}
+                                    errorObject={error}
+                                />
+                            )}
+                            <Trans
+                                t={t}
+                                i18nKey="publicSubmissionPrompt"
+                                components={{
+                                    b: <b />,
+                                    br: <br />,
+                                    support: (
+                                        <Link
+                                            href="#"
+                                            component="button"
+                                            onClick={(event) => {
+                                                // prevent form submission
+                                                event.preventDefault();
+                                                intercomShow();
+                                            }}
+                                        />
+                                    ),
```

Comment:
This `Link` could probably use a `variant="body2"` prop, so that the `button` font matches the style of the surrounding text.